modules/persistent-storage-csi-efs-metrics-procedure-cli.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+[id="efs-metrics-procedure-cli_{context}"]
+= Enabling usage metrics using the CLI
+
+To enable AWS EFS storage CSI usage metrics using the CLI:
+
+. Edit the `ClusterCSIDriver` YAML file by running the following command:
++
+[source, terminal]
+----
+$ oc edit clustercsidriver efs.csi.aws.com -o yaml
+----
+
+. Under `spec.aws.efsVolumeMetrics.state`, change the value from `Disabled` (default) to `RecursiveWalk`.
++
+`RecursiveWalk` indicates that volume metrics collection in the AWS EFS CSI Driver is performed by recursively walking through the files in the volume.
+
+. (Optional) To define how the recursive walk operates, you can also set the following fields:
++
+** `refreshPeriodMinutes`: Specifies the refresh frequency for volume metrics in minutes. If this field is left blank, a reasonable default is chosen, which is subject to change over time. The current default is 240 minutes. The valid range is 1 to 43,200 minutes.
+** `fsRateLimit`: Defines the rate limit for processing volume metrics in goroutines per file system. If this field is left blank, a reasonable default is chosen, which is subject to change over time. The current default is 5 goroutines. The valid range is 1 to 100 goroutines.
+
+. Save the changes to the YAML file.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+. Click *Administration* > *CustomResourceDefinitions*.
+
+. On the *CustomResourceDefinitions* page next to the *Name* dropdown box, type "clustercsidriver".
+
+. Click *CRD ClusterCSIDriver*.
+
+. Click the *YAML* tab.
+
+. Under `spec.aws.efsVolumeMetrics.state`, change the value from `Disabled` (default) to `RecursiveWalk`.
++
+`RecursiveWalk` indicates that volume metrics collection in the AWS EFS CSI Driver is performed by recursively walking through the files in the volume.
+
+. (Optional) You can also set the following fields that determine how the recursive walk operates:
```

Comment:
Per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks):
```suggestion
. Optional: You can also set the following fields that determine how the recursive walk operates:
```